src/seth/default.nix

```diff
@@ -18,6 +18,10 @@ stdenv.mkDerivation rec {
         else ""}
   '';
 
+  postFixup = ''
```

Comment:
This should probably be `postInstall` and the current `postInstall` should be `postFixup`.

Reply:
Xtuocgf bpda xopn